docs/concepts/privacy.md

```diff
@@ -0,0 +1,63 @@
+
+### Content identifiers
+
+Because IPFS uses [content addressing](/concepts/content-addressing/) rather than the legacy web's method of location addressing, each piece of data stored in the IPFS network gets its own unique content identifier (CID). Copies of the data associated with that CID can be stored in any number of locations worldwide on any number of participating IPFS nodes. To make retrieving the data associated with a particular CID efficient and fast, IPFS uses a [distributed hash table](http://localhost:8080/concepts/dht/) (DHT) to keep track of what's stored where. When you use IPFS to retrieve a particular CID, your node queries the DHT to find the closest nodes to you with that item — and by default also agrees to reprovide that CID to other nodes for a limited time, until periodic "garbage collection" clears your cache of content you haven't used in a while. You can also "pin" CIDs that you want to make sure are never garbage-collected, which also means you're acting as a permanent reprovider of that data.
+
+This is one of the advantages of IPFS over traditional legacy-web hosting. It means retrieving files — especially popular ones that exist on lots of nodes in the network — can be faster and more bandwidth-efficient. However, it's important to note that those DHT queries happen in public. Because of this, it's possible that third parties could be monitoring this traffic to determine what CIDs are being requested, when, and by whom. As IPFS continues to grow in popularity, it's more likely that such monitoring will exist.
+
+### Node IDs and node requests
+
+The other half of the equation when considering the prospect of IPFS traffic monitoring is that nodes' unique identifiers are themselves public. Just like with CIDs, every individual IPFS node has its own public identifier, such as `QmRGgYP1P5bjgapLaShMVhGMSwGN9SfYG3CM2TfhpJ3igE`.
+
+While a long string of letters and numbers may not be a "Johnny Appleseed" level of human-readable specificity, it's still a long-lived, unique identifier for your node. Keep in mind that it's possible to do a DHT lookup on your node's identifier and, particularly if your node is regularly running from the same location (like your home), find your IP address. Additionally, longer-term monitoring of the public IPFS network could yield information about what CIDs your node is requesting and/or reproviding and when.
+
+## Enhancing your IPFS privacy
+
+If there are situations in which you know you'll need to remain private but still want to use IPFS, one of the approaches outlined below may help. And don't forget, you can always discuss privacy and get others' input or ideas in the official [IPFS forums](https://discuss.ipfs.io).
+
```

Comment:
Useful to mention that one can tweak default provider setting of their node:

```suggestion

### Disable reproviding

By default IPFS node will announce itself as a provider for every CID in its cache. One can change that behavior by adjusting [reprovider settings](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#reprovider) in node's config.

Be aware that other peers won't be able to find the content if nobody is providing it, so make sure you pin and provide content that you care about.

```

Reply:
Excellent point. Added this in https://github.com/ipfs/ipfs-docs/pull/554/commits/1cd6c4ddbfc5154d5beeb09490e9843a20c478c2.